### ISSUE TITLE
Add the ability to disabile a site at all

### ISSUE THREAD

**Issue by marcominerva** [marcominerva/TinyCMS#33]
It is necessary to add an `IsPublished` column to the **Sites** table so that we can easily enable/disable the entire site rather than a single page. If a site isn't published, each request should return 404.

**Comment by N1K0232** [comment]
If you need, I can take care of this issue. In this case the 'IsPublished' column will work like a entity framework query filter right?

**Comment by N1K0232** [comment]
Yeah I know. My bad. I was meaning that is like it uses entity framework, because of the IsPublished property.
Sorry for the inconvenience

**Comment by marcominerva** [comment]
Ok, so I have assigned the issue to you. Thank!
